lms/templates/discovery/error.underscore

```diff
@@ -0,0 +1 @@
+<%= gettext("There was an error, try searching again.") %>
```

Comment:
This seems too small to require an underscore template.  Why not just call `gettext()` directly in the view?